lib/src/models/data/schedule.dart

```diff
@@ -73,10 +72,22 @@ class ScheduleModel extends Model {
 					day == null ? null : Day.fromJson(day)
 			]
 		];
+
 		setToday();
 		notifyListeners();
 	}
 
+	///
+	Future<Map<String, Schedule>> getDefaultSchedules() async {
+		for(MapEntry<String, String> entry in await
+		(Services.instance.database.calendar.cloud.getDefaultSchedules()).entries){
```

Comment:
```suggestion
		(Services.instance.database.calendar.getDefaultSchedules()).entries){
```

No need to specify `cloud` here, since we support offline mode (both for the user's convenience and to save reads on Firestore).